assets/scss/components/_goal-summary-card.scss

```diff
@@ -37,6 +37,13 @@
       padding-bottom: govuk-spacing(2);
       box-shadow: inset 0 -1px $govuk-border-colour;
     }
+
+    &--empty-no-shadow {
```

Comment:
OK I'll bite - do you want to show me the difference between these two uses tomorrow?

Reply:
Will do.